src/main/java/org/radarbase/management/config/LoggingConfiguration.kt

```diff
@@ -0,0 +1,37 @@
+package org.radarbase.management.config
+
+import ch.qos.logback.classic.LoggerContext
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+import tech.jhipster.config.JHipsterProperties
+import tech.jhipster.config.logging.LoggingUtils
+
+@Configuration
+open class LoggingConfiguration(
+    @Value("\${spring.application.name}") appName: String,
+    @Value("\${server.port}") serverPort: String,
+    jHipsterProperties: JHipsterProperties,
+    mapper: ObjectMapper
+) {
+    /** Logging configuration for JHipster.  */
+    init {
+        val context = LoggerFactory.getILoggerFactory() as LoggerContext
+        val map: MutableMap<String, String> = HashMap()
```

Comment:
Use buildMap instead for nicer syntax and mutability guarantees. For just two keys, you can also use mapOf